save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -55,13 +56,26 @@ class VulnerabilityController(
         vulnerabilityService.getFiltered(filters, authentication).map(Vulnerability::toDto)
     }
 
-    @GetMapping("/by-name-with-description")
+    @GetMapping("/by-name")
     @Operation(
         method = "GET",
         summary = "Get vulnerability by name.",
         description = "Get vulnerability by name.",
     )
     @ApiResponse(responseCode = "200", description = "Successfully fetched vulnerability by name")
+    fun getVulnerabilityByName(
+        @RequestParam name: String,
+    ): Mono<VulnerabilityDto> = blockingToMono {
+        vulnerabilityService.findByName(name)?.toDto()
+    }.switchIfEmptyToNotFound()
+
+    @GetMapping("/by-name-with-description")
```

Comment:
Sounds a bit confusing. Is it by `name` with `description` or with `description` by `name`?

Reply:
Entity with description by name.
But this is an old method. I didn’t add it now, I don’t know why GitHub singled it out.